### ISSUE TITLE
AI Hounds Tooth Title - crashes game on nashtah pup launch 

### ISSUE THREAD

**Issue by LordSkritch** [Baledin/FlyCasual#37]
** Fly Casual Release Version**
[Version 2.5 - 2.5.20240223.03](https://github.com/Baledin/FlyCasual/releases/tag/2.5.20240223.03)

**Describe the bug**
When AI Yv-666 with Hound's Tooth title dies and nashtah pup should launch. Game hangs.

**To Reproduce**
Pick YV-666,
Select title - Hound Tooth
Add additional ship - Nashtah Pup.
Play game against this list with AI
When you destroy YV-666 game hangs.

**Expected behavior**
Nashtah Pup should launch front or rear then YV-666 is removed.

**Screenshots**

**Platform/Device (please complete the following information):**
Windows 11

**Additional context**
Had this problem for quite a few versions now

**Comment by grandpajive** [comment]
Quick note, this happens with a docked shuttle in the VCX and the escape craft with Landos Millennium Falcon.

My workaround is to modify the custom ai saves and remove the title.

**Comment by Baledin** [comment]
Can you confirm it crashes? I'm testing this and not getting a crash (which usually has an error), instead it just doesn't continue at all. It appears that it's supposed to be removing the previous ship and returning control to the game but that's not happening.

**Comment by grandpajive** [comment]
You're right, the game doesn't crash. No further actions within the game are available. 

I've tried to enter debug and continue on event but it will not finish and allow you to continue

**Comment by LordSkritch** [comment]
Software stops responding = crash

**Comment by Baledin** [comment]
I realized something while testing this today, it doesn't appear to actually crash or freeze but the user input isn't obvious or user friendly. A maneuver selection appears, selecting a maneuver will move the Pup and continue the game as normal. Please try this out on your end and let me know if you still have an issue or if this is indeed just not a clear step in the process. I will try to think of ways to make this more obvious to the player. Perhaps a popup indicating that the user needs to select a maneuver for the escape craft?

**Comment by Baledin** [comment]
Please let me know if the above has resolved your immediate issue or if you see something different. I will close this issue on January 1 if no response. Thanks!

**Comment by Baledin** [comment]
Closing as I do not believe that this is a bug and no confirmation of an issue has been received.
